learn/classes/import.md

```diff
@@ -148,4 +152,40 @@ Suppose we have a package called `example` that contains a submodule `utils` wit
 # obj = MyUtils()
 ```
 
-After importing the class, we can create an instance of it by calling the constructor with parentheses.  
+After importing the class, we can create an instance of it by calling the constructor with parentheses.
+
+## Importing Classes from a Parent Directory with `sys.path`
+
+The `sys` module in Python provides access to various system-specific parameters and functions. The `sys.path` list is one of the attributes in this module and contains the directories where Python searches for modules. To import classes or modules from a directory outside the default search path, you can manipulate this list.
+
+Here's a step-by-step guide to importing all classes from another folder in the parent directory:
+
+1. Identify the Target Directory: First, you need to identify the directory from which you want to import classes. This directory should be in the parent directory of your current script.
```

Comment:
Let's switch to H3 headings or highlight `Identify the Target Directory` text with bold. + Further items as well.